lib/locales/en-KE.yml

```
@@ -0,0 +1,281 @@
+en-KE:
+  faker:
+    internet:
+      domain_suffix:
+        - ke
+        - com
+        - co.ke
+        - org.ke
+        - or.ke
+        - net
+        - go.ke
+        - biz
```

```suggestion
        - go.ke
```

similar to above's comment.